### PR TITLE
Make Gcloud installation optional

### PR DESCRIPTION
**Pull request by sjcqs**
Google Cloud SDK weights around *0,62 Go* on the expanded image

Using `docker images` locally: 
> The SIZE is the cumulative space taken up by the image and all its parent images. This is also the disk space used by the contents of the Tar file created when you docker save an image.

| REPOSITORY     |      TAG      |           IMAGE ID  |    SIZE |
|----|----|----|----|
| fabernovel/android |   api-29-ndk |          cf1b08589bfb        |     5.22GB |
| fabernovel/android |   api-29-gcloud-ndk |  8061f761b350        |  5.84GB |
| fabernovel/android |  api-29-gcloud      | 085ff6890d3b        |    1.97GB |
| fabernovel/android |  api-29             | 3f129b753e4d           | 1.35GB |

### CODE REVIEW

**Comment by sjcqs** [comment]
Fixes #68

File: ci_cd.sh [hunk 1]
```diff
@@ -59,12 +62,15 @@ if [ $large_test = true ]; then
...
+org_name="fabernovel"
+simple_image_name="api-$android_api"
+if [[ "$gcloud" = true ]]; then
+  simple_image_name="$simple_image_name-gcloud"
+fi
 if [ "$android_ndk" = true ]; then
   simple_image_name="$simple_image_name-ndk"
 fi
-branch=${GIT_REF##refs/heads/}
+branch="${GIT_REF##refs/heads/}"
```
Comment (vincentbrison) [inline ci_cd.sh:73]: Is this change needed? For best practice perhaps?

Comment (sjcqs) [reply ci_cd.sh:73]: https://google.github.io/styleguide/shellguide.html#quoting

Comment (vincentbrison) [reply ci_cd.sh:73]: Great guide! But we must be sure its ok with `#!/bin/sh` (the guide is for `bash`).

I should open an issue to see at which point we can go back to bash (was something related with GitHub Action container, but it might be obsolete now).

Comment (vincentbrison) [reply ci_cd.sh:73]: #70
